package.json

```diff
@@ -52,6 +52,7 @@
     "qr-image": "^3.2.0",
     "qrcode": "^1.5.3",
     "reflect-metadata": "^0.1.13",
+    "reload": "^3.2.1",
```

Comment:
@YoginiTayade  why this new package install? what is use of that?

Reply:
resolved